### PR TITLE
cyber markets #101 (GDAX exchange integration)

### PR DESCRIPTION
**Pull request by mgergalov**
- add trades from Gdax;
- add orders from Gdax;
- create cm-connectors-gdax docker image;
- update circle ci;
- upd docker-composes.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -168,6 +168,20 @@ project(":connectors-poloniex") {
...
 }
 
+project(":connectors-gdax") {
+    apply plugin: "application"
+
+    dependencies {
+        compile(project(":connectors-core"))
+    }
+
+    mainClassName = "fund.cyber.markets.connectors.poloniex.GdaxConnectorApplicationKt"
```
Comment (hleb-albau) [inline build.gradle]: package poloniex ?

File: connectors-core/src/test/kotlin/fund/cyber/markets/exchanges/gdax/GdaxMessageParserTest.kt [hunk 2]
```diff
@@ -0,0 +1,79 @@
+package fund.cyber.markets.exchanges.bitfinex
```
Comment (hleb-albau) [inline connectors-core/src/test/kotlin/fund/cyber/markets/exchanges/gdax/GdaxMessageParserTest.kt]: wrong package

File: connectors-core/src/test/kotlin/fund/cyber/markets/exchanges/gdax/GdaxMessageParserTest.kt [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+import fund.cyber.markets.connectors.bitfinex.connector.BitfinexTradesMessageParser
```
Comment (hleb-albau) [inline connectors-core/src/test/kotlin/fund/cyber/markets/exchanges/gdax/GdaxMessageParserTest.kt]: unused imports

File: devops/development/cybermarkets.yaml [hunk 4]
```diff
@@ -0,0 +1,674 @@
...
+    creationTimestamp: null
+    labels:
+      io.kompose.service: cn-kafka-claim0
+    name: cn-kafka-claim0
+  spec:
+    accessModes:
+    - ReadWriteOnce
+    resources:
+      requests:
+        storage: 100Mi
```
Comment (hleb-albau) [inline devops/development/cybermarkets.yaml]: 100mb for kafka?

File: build.gradle (hunk 1 above)
Comment (mgergalov) [reply build.gradle]: fix

File: connectors-core/src/test/kotlin/fund/cyber/markets/exchanges/gdax/GdaxMessageParserTest.kt (hunk 2 above)
Comment (mgergalov) [reply connectors-core/src/test/kotlin/fund/cyber/markets/exchanges/gdax/GdaxMessageParserTest.kt]: fix

File: connectors-core/src/test/kotlin/fund/cyber/markets/exchanges/gdax/GdaxMessageParserTest.kt (hunk 3 above)
Comment (mgergalov) [reply connectors-core/src/test/kotlin/fund/cyber/markets/exchanges/gdax/GdaxMessageParserTest.kt]: fix

File: devops/development/cybermarkets.yaml (hunk 4 above)
Comment (mgergalov) [reply devops/development/cybermarkets.yaml]: fix

**Pull request merged**